app/controllers/concerns/announce_actions/announce_filter.rb

```diff
@@ -39,6 +42,7 @@ def announce_resource(action, controller)
     def decorator_class(controller)
       return options[:decorator] if options[:decorator]
       resource_class = controller.controller_name.singularize.camelize
+      "Api::Msg::#{resource_class}Representer".safe_constantize ||
```

Comment:
Yeah, it's probably going to be just audio and images.  The explicit would probably be better in this case.  Will update!